### ISSUE TITLE
Can't delete the chronology

### ISSUE THREAD

**Issue by LOLLYTHECAT** [mozillahispano/openwapp#68]
I can't delete the chronology with a contact history reappears after a while...

FFOS 2.0

**Comment by bobcanthelpyou** [comment]
@Trynight could you still reproduce this issue with the latest v1.8.1 release of Openwapp?

**Comment by Trynight** [comment]
![2015-08-08-12-56-52](https://cloud.githubusercontent.com/assets/4656862/9150261/dda6f096-3dcf-11e5-860a-b031f1ab6dca.png)
![2015-08-08-12-56-46](https://cloud.githubusercontent.com/assets/4656862/9150262/dda866ba-3dcf-11e5-8925-fa6134e69cef.png)

Ultimate version of OpenWapp...

**Comment by bobcanthelpyou** [comment]
How do you delete your chat history? Go to your Inbox and keep touching a chat or a group to delete it. (longer then for enter a chat) and you will see a prompt to delete it completely.

![png image 320 x 539 pixels](https://cloud.githubusercontent.com/assets/4523264/9150362/c211f82a-3dd5-11e5-8951-6694e0a0a845.png)

**Comment by Trynight** [comment]
I can't delete single message...only all... is little annoying

**Comment by bobcanthelpyou** [comment]
It does not work to 'touch' a single message for a while until the following message appear and to delete it?

![png image 320 x 539 pixels](https://cloud.githubusercontent.com/assets/4523264/9150851/63ddcadc-3ded-11e5-82ee-bef72a877851.png)

Could you describe a little bit more detailed what the issue is you running into? 
> Can't delete the chronology
doesn't sound like 
> I can't delete single message...only all

Is your problem possibly described at issue #95?

**Comment by Trynight** [comment]
Yeah, i can't delete the single messages.. they duplicated themself and the chat take really much time to load

**Comment by Timmaks** [comment]
@Trynight do you still have this issue in the current version?
